src/stores/Binding/types.ts

```diff
@@ -145,7 +152,8 @@ export interface BindingState
     // and bindings are added to the specification via the collection selector.
     prefillResourceConfigs: (
         targetCollections: string[],
-        disableOmit?: boolean
+        disableOmit?: boolean,
+        trackAddition?: boolean
```

Comment:
`trackAddition` seems to always be getting passed as `true`... we might not need to pass it anymore?

Reply:
I was on the fence as to whether it is _safe_ to let this be the default behavior of `prefillResourceConfigs`. It definitely can be removed today, but will we forget this behavior exists in the future if removed.

Reply:
This input argument was removed from this store action.